packages/example-app-base/src/components/Page/sections/BestPracticesSection.tsx

```diff
@@ -18,79 +18,70 @@ export const BestPracticesSection: React.StatelessComponent<IBestPracticesSectio
     fileNamePrefix,
     componentUrl,
     platform,
-    sectionName = 'Best Practices',
+    sectionName,
+    readableSectionName,
     bestPractices,
     dos,
     donts,
     style,
+    id,
     title = 'Page'
   } = props;
-  const { readableSectionName = sectionName } = props;
-  const sectionId = pascalize(sectionName);
   const bestPracticesUrl = componentUrl
-    ? getEditUrl({ name: fileNamePrefix || title, section: sectionId, baseUrl: componentUrl, platform })
+    ? getEditUrl({ name: fileNamePrefix || title, section: pascalize(sectionName!), baseUrl: componentUrl, platform })
     : undefined;
   const dosUrl = componentUrl ? getEditUrl({ name: fileNamePrefix || title, section: 'Dos', baseUrl: componentUrl, platform }) : undefined;
   const dontsUrl = componentUrl
     ? getEditUrl({ name: fileNamePrefix || title, section: 'Donts', baseUrl: componentUrl, platform })
     : undefined;
 
-  const dosAndDonts: JSX.Element[] = [];
-
-  dosAndDonts.push(
-    <div className={styles.subSection} key="best-practices">
-      <div className={styles.sectionHeader}>
-        <h2 className={styles.subHeading} id={sectionId}>
-          {readableSectionName}
-        </h2>
-        {bestPractices && bestPracticesUrl && (
-          <EditSection className={styles.edit} title={title} section="Best Practices" url={bestPracticesUrl} />
-        )}
-      </div>
-      {bestPractices && (
-        <div className={styles.content}>
-          <Markdown>{bestPractices}</Markdown>
+  return (
+    <div className={className} style={style}>
+      <div className={styles.subSection}>
+        <div className={styles.sectionHeader}>
+          <h2 className={styles.subHeading} id={id}>
+            {readableSectionName || sectionName}
+          </h2>
+          {!!(bestPractices && bestPracticesUrl) && (
+            <EditSection className={styles.edit} title={title} section="Best Practices" url={bestPracticesUrl} />
```

Comment:
"Best practices" ?

Reply:
Right now we're *wildly* inconsistent with casing, and this one has historically been title case.